desktop/mac/release-notes/index.md

```diff
@@ -17,14 +17,47 @@ redirect_from:
 
 This page contains information about the new features, improvements, known issues, and bug fixes in Docker Desktop releases.
 
-## Docker Desktop 4.1.1
-2021-10-12
+## Docker Desktop 4.2.0
+2021-11-08
 
 > Download Docker Desktop
 >
 > [Mac with Intel chip](https://desktop.docker.com/mac/main/amd64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-amd64){: .button .primary-btn }
 > [Mac with Apple chip](https://desktop.docker.com/mac/main/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){: .button .primary-btn }
 
+### New
+
+- **Pause/Resume**: You can now pause your Docker Desktop session when you are not actively using it and save CPU resources on your machine. For more information, see [Pause/Resume](../index.md#pauseresume).
+- **Software Updates**: The option to turn off automatic check for updates is now available for users on all Docker subscriptions, including Docker Personal and Docker Pro. All update-related settings have been moved to the **Software Updates** section. For more information, see [Software updates](../index.md#software-updates).
+- **Window management**: The Docker Dashboard window size and position persists when you close and reopen Docker Desktop.
+
+### Upgrades
+
+- [Docker Engine v20.10.10](https://docs.docker.com/engine/release-notes/#201010)
+- [containerd v1.4.11](https://github.com/containerd/containerd/releases/tag/v1.4.11)
+- [runc v1.0.2](https://github.com/opencontainers/runc/releases/tag/v1.0.2)
+- [Go 1.17.2](https://golang.org/doc/go1.17)
+- [Compose CLI v2.1.1](https://github.com/docker/compose-cli/tree/v1.0.21)
```

Comment:
```suggestion
- [Compose v2.1.1](https://github.com/docker/compose/releases/tag/v2.1.1)
```

Reply:
Yes, I don’t think we’ve documented compose-cli releases in the past.